lib/_constants.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   CLASS_NAMESPACE: 'gmusic-theme',
+  CLASS_NAMESPACE_LIGHT: 'gmusic-theme-light',
```

Comment:
If we are going to go with light or highlight, we should be consistent. Please name this `CLASS_NAMESPACE_HIGHLIGHT` and `gmusic-theme-highlight`